scripts/lib/install

```diff
@@ -539,3 +539,7 @@ fi
 su zulip -c '/home/zulip/deployments/current/scripts/setup/initialize-database --quiet'
 
 su zulip -c '/home/zulip/deployments/current/manage.py generate_realm_creation_link'
+
+# setting up the gitconfig
+git config --global user.email "$ZULIP_ADMINISTRATOR"
+git config --global user.name "$EXTERNAL_HOST"
```

Comment:
We should check if these settings are already set rather than overwriting them.

Reply:
@timabbott  please correct me if I am wrong... 
we run the script when we create a new environment.. so I assumed that since it is a fresh installation, we can just set those configs... is that wrong?? should I check if a `.gitconfig` file already exists, and if it doesn't then run these commands??

Reply:
This looks like it's setting up a `.gitconfig` for the `root` user, not the `zulip` user.  100% agree that the `zulip` user can't have existed beforehand, so it's fine to overwrite; but we shouldn't modify `/root/.gitconfig` if it exists.

But I think the issue was about setting up a `.gitconfig` for the `zulip` user.  Swap this around to do that, and it should be fine.

Reply:
Thanks for clarifying that @alexmv!
just wanted to clarify, the config has to be created for the zulip user, that is, we should use `git config` with `--local` flag, so that the config file is in the `.git` directory of the zulip repo?

Reply:
No, because every `~zulip/deployments/current/` is a new directory each deploy.  If we store it in those directories, it would only apply fir the first deploy.

We need to run `git config --global` _as the `zulip` user_.